PyInstaller/hooks/hook-win32ctypes.core.py

```diff
@@ -0,0 +1,26 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2020, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License (version 2
+# or later) with exception for distributing the bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#
+# SPDX-License-Identifier: (GPL-2.0-or-later WITH Bootloader-exception)
+#-----------------------------------------------------------------------------
+
+# TODO: remove this hook during PyInstaller 4.5 release cycle!
```

Comment:
Anyhow, I'm afraid, nobody will search for this type of comment when releasing PyInstaller 4.5,

Anyway: What does this comment mean? Why for PyInstaller 4.5?

Reply:
I'm not sure, either. @Legorooj asked me to place a copy of this hook in both main repository and hooks-contrib one, and add a note about removal in version 4.5 to this copy. Perhaps he can explain?

Reply:
We need the hook in here because it's required by the keyring hook, which we haven't moved out of PyInstaller yet. The note basically tells us to remove it in a couple of versions when the hook should, in theory, not be used from PyInstaller anymore, but contrib. If you have a better way to deal with this I'm all ears?

Reply:
IC, indeed this is a good idea. I have no better idea. so let's see how this will work.